Bottom: aed932240623fe68d64cafd934bce354a91d1031
Top:    aad0eb0a160fdf4a6b51a39cab2586aa355a5a0a
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 13:29:38 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/lib/master_tok.l b/lib/master_tok.l
index dfdcb14..6ff18c0 100644
--- a/lib/master_tok.l
+++ b/lib/master_tok.l
@@ -91,8 +91,7 @@ OPTWS		[[:blank:]]*
 NL		\r?\n
 CONT		\\\n{OPTWS}
 
-OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=([[:alnum:]_\-])+)*)+)
-QVOPTIONSTR	([[:alpha:]_]([[:alnum:]_\-])*="([[:alnum:]_\-\:])+")
+OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(="?([[:alnum:]_\-\:])+\"?)*)+)
 MACROSTR	(-D{OPTWS}([[:alpha:]_]([[:alnum:]_\-\.])*)=([[:alnum:]_\-\.])+)
 SLASHIFYSTR	(--(no-)?slashify-colons)
 NUMBER		[0-9]+
@@ -401,11 +400,6 @@ OPTNTOUT	(-n{OPTWS}|-n{OPTWS}={OPTWS}|--negative-timeout{OPTWS}|--negative-timeo
 		return(OPTION);
 	}
 
-	{QVOPTIONSTR} {
-		strcpy(master_lval.strtype, master_text);
-		return(OPTION);
-	}
-
 	{OPTIONSTR} {
 		strcpy(master_lval.strtype, master_text);
 		return(OPTION);
